src/Language/Haskell/Liquid/Types.hs

```diff
@@ -458,8 +458,9 @@ instance Hashable (PVar a) where
 ------ Strictness --------------------------------------------------
 --------------------------------------------------------------------
 
-instance NFData Var
-instance NFData SrcSpan
+instance NFData Var where rnf _ = ()
```

Comment:
Unfortunately not, because standalone deriving requires the data constructors to be in scope, which they are not (at least for `Var`).